charts/xui-mo-webapp/values.yaml

```diff
@@ -45,7 +45,9 @@ nodejs:
     S2S_SERVICE: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     FEE_AND_PAY_API: http://payment-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     SERVICES_TERMS_AND_CONDITIONS_API_SERVICE: http://xui-terms-and-conditions-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
-  keyVaults:
+    MANAGE_CASE_LINK: https://manage-case.platform.hmcts.net/cases
+    MANAGE_ORG_LINK: https://manage-org.platform.hmcts.net
+    keyVaults:
```

Comment:
Maintain the indentation around keyVaults: as indentation matters in .yaml files.